src/ground_truth_nsrts.py

```diff
@@ -400,7 +411,7 @@ def grasp_sampler(state: State, rng: np.random.Generator,
         }
         add_effects = {LiftedAtom(HandEmpty, [])}
         delete_effects = {LiftedAtom(Holding, [can])}
-        place_sampler = lambda s, r, o: np.array([
+        place_sampler = lambda s, g, r, o: np.array([
```

Comment:
wasn't the goal of this PR to make a goal-conditioned version of the place sampler? is that coming in a separate PR?

Reply:
yeah that will happen separately